### PR TITLE
Added exercism completion

### PR DESCRIPTION
**Pull request by mhdali**
## Description

I added `exercism` completion file.

For more info about `exercism` visit http://exercism.io/

### CODE REVIEW

File: share/completions/exercism.fish [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                        return 1
+                end
+        end
+        return 0
+end
+
+complete -c exercism -s c -l config -d "path to config file [$EXERCISM_CONFIG_FILE, $XDG_CONFIG_HOME]"
+complete -c exercism -s v -l verbose -d "turn on verbose logging"
+complete -c exercism -s h -l help -d "show help"
+complete -c exercism -s v -l version -d "turn on verbose logging"
```
Comment (floam) [inline share/completions/exercism.fish]: Are you sure about this?

Comment (mhdali) [reply share/completions/exercism.fish]: You're right, will fix it.

**Comment by mhdali** [comment]
@floam Anything else?

File: share/completions/exercism.fish [hunk 2]
```diff
@@ -0,0 +1,27 @@
+function __fish_exercism_no_subcommand -d 'Test if exercism has yet to be given the subcommand'
+        for i in (commandline -opc)
+                if contains -- $i demo debug configure fetch restore submit unsubmit tracks download help
+                        return 1
+                end
+        end
+        return 0
+end
+
+complete -c exercism -s c -l config -d "path to config file [$EXERCISM_CONFIG_FILE, $XDG_CONFIG_HOME]"
```
Comment (faho) [inline share/completions/exercism.fish]: These variables will be expanded when the completion is loaded. Do you want that?

**Comment by faho** [comment]
Merged with squash (and fixing those quotes) as ba04c2af3db927deebb672a176d0c5de53311341.

Thanks, and sorry for the delay!

**Pull request closed without merging**